stock_dynamic_routing/tests/test_routing_pull.py

```diff
@@ -0,0 +1,911 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+
+from odoo.tests import common
+
+
+class TestRoutingPullCommon(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
on all classes possibly